src/pages/Stream/utils.ts

```diff
@@ -1,17 +1,42 @@
 import { Log } from '@/@types/parseable/api/query';
 import { LogStreamSchemaData } from '@/@types/parseable/api/stream';
 import { columnsToSkip } from './providers/LogsProvider';
+import { parseSQL } from 'react-querybuilder';
+import { QueryType, RuleGroupTypeOverride, RuleTypeOverride } from './providers/FilterProvider';
 
 export const getPageSlice = (page = 1, perPage: number, data: Log[]) => {
 	const firstPageIndex = (page - 1) * perPage;
 	const lastPageIndex = firstPageIndex + perPage;
 	return data ? data.slice(firstPageIndex, lastPageIndex) : [];
 };
 
+export const parseSQLWithIDs = (sqlString: string) => {
```

Comment:
this variable name can be misleading. Please change it to a more appropriate name

Reply:
Addressed this